Makefile

```diff
@@ -6,7 +6,7 @@
 TOP = .
 include $(TOP)/build.mk
 
-SUB_DIRS = utils ercc ecc_enclave ecc tlcc_enclave tlcc examples integration # docs
+SUB_DIRS = utils ercc ecc_enclave ecc tlcc_enclave tlcc examples integration demo # docs
```

Comment:
I would not include demo to the toplevel make at the moment - we maybe promote this in the future to a dedicated repo; or move it to examples or it just stays here. But I still think that a `demo/Makefile` makes sense.

Reply:
or if we add it to the top level make; let's build the demo bu just calling `make demo` but not including it into `make`.

Reply:
Hmm, not sure why eventually outsourcing demo (with which i agree, although as discussed in issue #199 we might also keep some stuff in generalized form in the main repo ...) should have any implication on building or not?   My motivation was two-folds:
- to make sure that demo artifacts are build when you build the overall repo and
- make sure travis also touches the demo on PRs, by at very least building it.
What is the problem of building demo in main build? It's not that it extends build time drastically or alike?  Also 'make demo' might actually be a better target to potentially (eventually) automatically set up the whole demo via a top-level make?

Reply:
I have mixed feelings about this: I agree with Marcus (yes, we discussed that); but also, as we update FPC, I would not like the demo to simply be "forgotten" and never work anymore because it's not maintained.
My conclusion is: (i) let's discuss if we should move it elsewhere, (ii) let's maintain it while it is in the repo (which is the good practice to follow).

Reply:
as discussed today we keep the demo in the repo and therefore also integrate it in the main make to ensure that travis has the chance to detect issues there.